docs/book/src/user/quick-start.md

```diff
@@ -747,603 +139,33 @@ ready.
 kubectl get clusters --output jsonpath="{range .items[*]} [{.metadata.name} {.status.ControlPlaneInitialized} {.status.ControlPlaneReady}] {end}"
 ```
 
-After the control plane node is up, we can retrieve the [workload cluster] Kubeconfig:
-
-{{#tabs name:"tab-getting-kubeconfig" tabs:"AWS/Azure/GCP/vSphere/OpenStack, Docker"}}
-{{#tab AWS/Azure/GCP/vSphere/OpenStack}}
+After the control plane node is up and ready, we can retrieve the [workload cluster] Kubeconfig:
 
 ```bash
-kubectl --namespace=default get secret/capi-quickstart-kubeconfig -o json \
-  | jq -r .data.value \
+kubectl --namespace=default get secret/capi-quickstart-kubeconfig -o jsonpath={.data.value} \
   | base64 --decode \
   > ./capi-quickstart.kubeconfig
 ```
 
-{{#/tab }}
-{{#tab Docker}}
+### Deploy a CNI solution
 
-```bash
-kubectl --namespace=default get secret/capi-quickstart-kubeconfig -o json \
-  | jq -r .data.value \
-  | base64 --decode \
-  > ./capi-quickstart.kubeconfig
-```
-
-When using docker-for-mac MacOS, you will need to do a couple of additional
-steps to get the correct kubeconfig:
-
-```bash
-# Point the kubeconfig to the exposed port of the load balancer, rather than the inaccessible container IP.
-sed -i -e "s/server:.*/server: https:\/\/$(docker port capi-quickstart-lb 6443/tcp | sed "s/0.0.0.0/127.0.0.1/")/g" ./capi-quickstart.kubeconfig
-
-# Ignore the CA, because it is not signed for 127.0.0.1
-sed -i -e "s/certificate-authority-data:.*/insecure-skip-tls-verify: true/g" ./capi-quickstart.kubeconfig
-```
-
-{{#/tab }}
-{{#/tabs }}
-
-Deploy a CNI solution, Calico is used here as an example.
-
-{{#tabs name:"tab-usage-addons" tabs:"Calico"}}
-{{#tab Calico}}
+Calico is used here as an example.
 
 ```bash
 kubectl --kubeconfig=./capi-quickstart.kubeconfig \
   apply -f https://docs.projectcalico.org/v3.8/manifests/calico.yaml
```

Comment:
```suggestion
  apply -f https://docs.projectcalico.org/v3.12/manifests/calico.yaml
```

3.12 has been tested for k8s 1.14 to 1.17 https://docs.projectcalico.org/v3.12/getting-started/kubernetes/requirements

Reply:
While 3.8 has been tested only up to 1.15 https://docs.projectcalico.org/v3.8/getting-started/kubernetes/requirements

Reply:
done!